cypress/integration/route_spec.js

```diff
@@ -0,0 +1,98 @@
+describe('Sea route tests', function () {
+
+    before (function () {
+	cy.login();
+    });
+
+    beforeEach(function () {
+        // Session cookies will not be cleared before the NEXT test starts
+        cy.preserveSessionOnce();
+    });
+
+
+    it ("Create new route", function () {
+	cy.visit ("/ote/#/routes/");
+
+	cy.wait(500);
+
+	cy.get('tr').should('have.length', 1);
+
+	cy.contains(/Lisää uusi merireitti/).click();
+    });
+
+    it ("Add basic information", function () {
+	cy.get('[id*="Reitinnimi"]').type("Test route").should('have.value', "Test route");
+	cy.get('[id*="Palveluntuottaja"]').click();
+	cy.contains(/Testinen/).click();
+	cy.get('[id*="departure-point-name"]').type("Oulu").should('have.value', "Oulu");
+	cy.get('[id*="destination-point-name"]').type("Vaasa").should('have.value', "Vaasa");
+	cy.get('[id*="undefined--Voimassaalkaen"]').click();
+	cy.get(':nth-child(3) > :nth-child(4) > span').click();
```

Comment:
Is is necessary to use nth-child? It will make our tests flaky. If the ordering of the elements change, you'll have to always update this assertion.

Reply:
We use material-ui calendar component here. So elements are most likely fixed.